.github/workflows/smoke-test-daily.yml

```diff
@@ -51,7 +51,7 @@ jobs:
             - name: Run 'Update WooCommerce' test.
               working-directory: plugins/woocommerce
               env:
-                  UPDATE_WC: true
+                  UPDATE_WC: nightly
```

Comment:
In this PR, `UPDATE_WC` will start accepting WooCommerce versions as the value, instead of just being a toggle. This means that we'll need to change its value in the "Smoke test nightly" workflow to `nightly` in order to not break it once this PR is merged.